modules/data-entry/src/main/frontend/src/questionnaire/PrintPreview.jsx

```diff
@@ -145,14 +157,16 @@ const PrintPreview = forwardRef((props, ref) => {
       </DialogContent>
       <DialogActions>
         <Button variant="contained" onClick={onClose}>Close</Button>
-        <Button variant="contained" color="primary" onClick={() => {window.print()}} disabled={!!!content}>Print</Button>
+        <Button variant="contained" onClick={handlePrint} disabled={!!!content}>Print</Button>
```

Comment:
`color="primary"` should not have been removed
```suggestion
        <Button variant="contained" color="primary" onClick={handlePrint} disabled={!!!content}>Print</Button>
```